src/frontend/App/context/BehandlingContext.tsx

```diff
@@ -56,8 +64,12 @@ const [BehandlingProvider, useBehandling] = constate(() => {
         );
     }, [vilkårsvurderinger]);
 
+    const erBehandlingRedigerbar = (behandling: Behandling): boolean =>
```

Comment:
Har dere satt opp Prettier i intellij? Hvis ikke kan det være lurt, så formaterer intellij alt for oss slik at vi slipper "endringer" pga ulik formateringsstrategi. :)